content/ko/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md

```diff
@@ -28,7 +28,7 @@ APIService를 구현하는 가장 일반적인 방법은 클러스터 내에 실
 Extension-apiserver는 kube-apiserver로 오가는 연결의 레이턴시가 낮아야 한다.
 kube-apiserver로 부터의 디스커버리 요청은 왕복 레이턴시가 5초 이내여야 한다.
 
-extention API server가 레이턴시 요구 사항을 달성할 수 없는 경우 이를 충족할 수 있도록 변경하는 것을 고려한다.
+extension API server가 레이턴시 요구 사항을 달성할 수 없는 경우 이를 충족할 수 있도록 변경하는 것을 고려한다.
```

Comment:
```suggestion
Extension API server가 레이턴시 요구 사항을 달성할 수 없는 경우 이를 충족할 수 있도록 변경하는 것을 고려한다.
```
@fenggw-fnst Thanks for your contribution. Let me correct one more thing. This is because the first letter of the sentence must be capitalized.

Reply:
Thanks for your review, done.